### PR TITLE
Feature/ge 605

### PR DESCRIPTION
**Pull request by AltusBarry**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,63 @@
+VENV=./ve
+PYTHON=$(VENV)/bin/python
+PIP=$(VENV)/bin/pip
+
+# Colours.
+CLEAR=\033[0m
```
Comment (cobusc) [inline Makefile]: Nifty

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	@echo "$(CYAN)Clearing virtualenv...$(CLEAR)"
+	rm -rf $(VENV)
+	@echo "$(GREEN)DONE$(CLEAR)"
+
+# Build sphinx docs, then move them to docs/ root for GitHub Pages usage.
+docs-build:
+	@echo "$(CYAN)Installing Sphinx requirements...$(CLEAR)"
+	$(PIP) install sphinx sphinx-autobuild
+	@echo "$(GREEN)DONE$(CLEAR)"
+	# Backup the files needed to build the html.
```
Comment (cobusc) [inline Makefile]: I guess with all the informational messages the comments are kinda redundant.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+	@echo "$(GREEN)DONE$(CLEAR)"
+	# Remove the tar file.
+	rm backup.tar
+	# Actually make html from index.rst
+	@echo "$(CYAN)Running sphinx command...$(CLEAR)"
+	$(MAKE) -C docs/ clean html
+	@echo "$(GREEN)DONE$(CLEAR)"
+	# Drop all build files in doc root.
+	@echo "$(CYAN)Moving build files to docs/ root...$(CLEAR)"
+	cp -r docs/build/html/. docs/
```
Comment (cobusc) [inline Makefile]: `docs/build/html/` (without the fullstop) is sufficient

File: Makefile (hunk 2 above)
Comment (AltusBarry) [reply Makefile]: I also thought so, but comments should I suppose not be influenced by any output you may or may not give?

File: Makefile (hunk 3 above)
Comment (cobusc) [inline Makefile]: Would a `rm -rf docs/build/` here help to keep things clean(er)?

Comment (AltusBarry) [reply Makefile]: It would locally, we don't push it to the repo. So I see no harm in removing it here.

File: Makefile (hunk 2 above)
Comment (cobusc) [reply Makefile]: Entirely your call on how to proceed.

**Comment by cobusc** [comment]
:+1:

File: Makefile (hunk 2 above)
Comment (AltusBarry) [reply Makefile]: Ok, I'll remove some of the obvious comments.

Comment (AltusBarry) [reply Makefile]: Which is almost all of them. Oh well.

File: Makefile (hunk 3 above)
Comment (AltusBarry) [reply Makefile]: Actually that places the html dir in root, we want the content in root. So it needs the fullstop.

**Pull request closed without merging**